packages/mcp-client/cli.ts

```diff
@@ -12,12 +13,13 @@ import { version as packageVersion } from "./package.json";
 const MODEL_ID = process.env.MODEL_ID ?? "Qwen/Qwen2.5-72B-Instruct";
 const PROVIDER = (process.env.PROVIDER as InferenceProvider) ?? "nebius";
 const BASE_URL = process.env.BASE_URL;
+const MCP_LOCAL_FOLDER = join(homedir(), ".mcp");
 
 const SERVERS: StdioServerParameters[] = [
 	{
 		// Filesystem "official" mcp-server with access to your Desktop
 		command: "npx",
-		args: ["-y", "@modelcontextprotocol/server-filesystem", join(homedir(), "Desktop")],
+		args: ["-y", "@modelcontextprotocol/server-filesystem", MCP_LOCAL_FOLDER],
```

Comment:
i would rather still use Desktop on mac OS and just `homedir()` on Win32, can you switch to this?

You can use `process.platform === "darwin"` to figure out 

Thanks!